.env.dusk.local

```diff
@@ -1,14 +1,14 @@
-APP_NAME="Meteostanice LKBASK - Hodoňovice"
+APP_NAME= "Meteostanice LKBASK - Hodoňovice"
```

Comment:
add this file to gitignore and prepare example file, now it contains your specific settings and it won't work on other machines

Reply:
created .env.dusk.local.example and added to .gitgnore file .env.dusk.local